qml/keys/NumKey.qml

```diff
@@ -100,12 +102,20 @@ Item {
 
     Text {
         id: keyLabel
-        text: (panel.activeKeypadState === "NORMAL") ? label : shifted;
-        anchors.centerIn: parent
+        //text: (panel.activeKeypadState === "NORMAL") ? label : shifted;
+		text: label
```

Comment:
I didn't realize the text for the num keys stayed the same in normal and shifted state. But I guess you did that on purpose :)